pages/FolderPage.py

```diff
@@ -14,20 +14,49 @@ class FolderPage(BasePage):
     def __init__(self, driver):
         super().__init__(driver)
 
+    special_characters = ['!', '@', '#', '$', '%', '^', '&', '*', '[', ']', '<', '>', '/', '\\', '|', ':', ';']
+    dots = ['.', '..']
+    three_or_more_dots = ['...', '....', '.....', '......', '.......', '........']
+    white_spaces = [' ', '  ', '   ', '    ', '     ', '      ', '       ', '        ']
+
```

Comment:
May be it will better use regular expressions?

Reply:
Underscore is not a special character, but it is possible to create a name in Jenkins like "_______________". Is this a bug? Or not?